<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>900</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(255,250,250)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="name_select">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>240</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
background-color: rgba(141,238,238,0.2);
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="name_label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>240</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
background-color: rgba(141,238,238,0);
}</string>
    </property>
    <property name="text">
     <string>多面体名字</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="aim_label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>300</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
background-color: rgba(141,238,238,0);
}</string>
    </property>
    <property name="text">
     <string>染色目标</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="rbw_label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>390</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
background-color: rgba(141,238,238,0);
}</string>
    </property>
    <property name="text">
     <string>【指定三种颜色组合】</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="aim_select">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>300</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
background-color: rgba(141,238,238,0.2);
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="color_label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>560</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
background-color: rgba(141,238,238,0);
}</string>
    </property>
    <property name="text">
     <string>【指定总颜色数】</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="b_label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>450</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
background-color: rgba(141,238,238,0);
}</string>
    </property>
    <property name="text">
     <string>蓝</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="w_label">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>450</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
background-color: rgba(141,238,238,0);
}</string>
    </property>
    <property name="text">
     <string>白</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="r_lable">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>450</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
background-color: rgba(141,238,238,0);
}</string>
    </property>
    <property name="text">
     <string>红</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="r_num">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>450</y>
      <width>50</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
background-color: rgba(141,238,238,0);
}</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="b_num">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>450</y>
      <width>50</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
background-color: rgba(141,238,238,0);
}</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="w_num">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>450</y>
      <width>50</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
background-color: rgba(141,238,238,0);
}</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="color_num">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>613</y>
      <width>50</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
background-color: rgba(141,238,238,0);
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="color_num_label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>610</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
background-color: rgba(141,238,238,0);
}</string>
    </property>
    <property name="text">
     <string>颜色总数</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1201</width>
      <height>127</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="title">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>多面体染色计数</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="subtitle">
       <property name="text">
        <string>2020组合数学作业</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QSplitter" name="splitter_2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>160</y>
      <width>450</width>
      <height>550</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="handleWidth">
     <number>20</number>
    </property>
    <widget class="QLabel" name="attr_label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>多面体属性</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="background2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
background-color: rgba(202,255,112,0.2);
border-radius:20px
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="picture">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>230</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
border: 2px solid rgba(205,92,92,0.5);
background-color: rgba(202,255,112,0);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="calculate">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>430</y>
      <width>100</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>计算</string>
    </property>
   </widget>
   <widget class="QLabel" name="member">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>810</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>小组成员</string>
    </property>
   </widget>
   <widget class="QLabel" name="function_descrip">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>810</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>功能说明</string>
    </property>
   </widget>
   <widget class="QLabel" name="develop_info">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>810</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>开发信息</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>160</y>
      <width>450</width>
      <height>550</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="handleWidth">
     <number>20</number>
    </property>
    <widget class="QLabel" name="para_label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="text">
      <string>参数选择</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="background1">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
background-color: rgba(141,238,238,0.2);
border-radius:20px
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="vertice_num">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>550</y>
      <width>200</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
border: 1px solid rgba(0,0,0,0.5);
background-color: rgba(202,255,112,0);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="edge_num">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>600</y>
      <width>200</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
border: 1px solid rgba(0,0,0,0.5);
background-color: rgba(202,255,112,0);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="surface_num">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>650</y>
      <width>200</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
border: 1px solid rgba(0,0,0,0.5);
background-color: rgba(202,255,112,0);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="result">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>730</y>
      <width>600</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
border: 2px solid rgba(0,0,0,0.5);
border-radius:20px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>390</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
background-color: rgba(141,238,238,0);
}</string>
    </property>
    <property name="text">
     <string>模式一</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>560</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
background-color: rgba(141,238,238,0);
}</string>
    </property>
    <property name="text">
     <string>模式二</string>
    </property>
   </widget>
   <zorder>splitter_2</zorder>
   <zorder>splitter</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>name_select</zorder>
   <zorder>name_label</zorder>
   <zorder>aim_label</zorder>
   <zorder>rbw_label</zorder>
   <zorder>aim_select</zorder>
   <zorder>color_label</zorder>
   <zorder>b_label</zorder>
   <zorder>w_label</zorder>
   <zorder>r_lable</zorder>
   <zorder>r_num</zorder>
   <zorder>b_num</zorder>
   <zorder>w_num</zorder>
   <zorder>color_num</zorder>
   <zorder>color_num_label</zorder>
   <zorder>picture</zorder>
   <zorder>calculate</zorder>
   <zorder>member</zorder>
   <zorder>function_descrip</zorder>
   <zorder>develop_info</zorder>
   <zorder>vertice_num</zorder>
   <zorder>edge_num</zorder>
   <zorder>surface_num</zorder>
   <zorder>result</zorder>
   <zorder>radioButton</zorder>
   <zorder>radioButton_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
